.clang-format

```diff
@@ -8,6 +8,7 @@ AllowShortIfStatementsOnASingleLine: false
 AllowShortLoopsOnASingleLine: false
 PointerAlignment: Left
 ColumnLimit: 80
+CommentPragmas: '[@\\].*'
```

Comment:
This is needed for clang-format to not break doxygen tags that are too long. This is needed for e.g. `@copydoc`.